test_py/test_access_tokens.py

```diff
@@ -0,0 +1,124 @@
+import time
+
+import pytest
+from src_py.grndwork_api_client import access_tokens
+from src_py.grndwork_api_client.make_request import make_request
+
+
+refresh_token = {
+    'subject': 'uuid',
+    'token': 'refresh_token',
+  }
+
+
+@pytest.fixture(name='make_request', autouse=True)
+def fixture_requests(mocker):
+    mockpatch = mocker.patch(
+        target='src_py.grndwork_api_client.access_tokens.make_request',
+        spec=make_request,
+    )
+    mockpatch.return_value = {'token': 'access_token'}, None
+    return mockpatch
+
+
+@pytest.fixture(name='decode', autouse=True)
+def fixture_decode(mocker):
+    mockdecode = mocker.patch(
+        target='src_py.grndwork_api_client.access_tokens.jwt.decode',
+    )
+    not_expired = int(time.time() * 1000) + 1000
+    mockdecode.return_value = {'exp': not_expired}
+    return mockdecode
+
+
+@pytest.fixture(autouse=True)
+def _clear_token_cache():
+    access_tokens.reset_access_token_cache()
+
+
+def describe_access_tokens():
+
+    def it_requests_new_access_token(make_request):
+        access_tokens.get_access_token(
+            refresh_token=refresh_token,
+            platform='platform',
+            scope='read:data',
+        )
+        (_, kwargs) = make_request.call_args
+
+        assert kwargs == {
+            'url': 'https://api.grndwork.com/v1/tokens',
+            'method': 'POST',
+            'body': {
+                'subject': 'uuid',
+                'platform': 'platform',
+                'scope': 'read:data',
+            },
+            'token': 'refresh_token',
+        }
+        assert make_request.call_count == 1
```

Comment:
```suggestion
        assert make_request.call_count == 1

        (_, kwargs) = make_request.call_args

        assert kwargs == {
            'url': 'https://api.grndwork.com/v1/tokens',
            'method': 'POST',
            'body': {
                'subject': 'uuid',
                'platform': 'platform',
                'scope': 'read:data',
            },
            'token': 'refresh_token',
        }
```

Asserting the call count before accessing the call args provides better error message when test fails

Reply:
Done